grafana-formula/grafana-formula.changes

```diff
@@ -1,8 +1,10 @@
 -------------------------------------------------------------------
 Thu Jun 26 11:26:48 UTC 2025 - Witek Bedyk <witold.bedyk@suse.com>
 
-- Version 0.12.1
+- Version 5.1.0
   * Add SLES 15 SP7 to supported versions (bsc#1245368)
+  * Drop Leap 42.3, SLES 12 SP3 and SLES 12 SP4 from supported
+    versions
```

Comment:
please do NOT re-edit already submitted changelog.

for 5.1, we  already had
~~~
-------------------------------------------------------------------
Thu Jun 26 11:26:48 UTC 2025 - Witek Bedyk <witold.bedyk@suse.com>

- Version 0.12.1
  * Add SLES 15 SP7 to supported versions (bsc#1245368)
~~~

For this new change with the version bump and the removal of old unsupported SLE versions, please add a new entry.

Also, 
`Drop Leap 42.3, SLES 12 SP3 and SLES 12 SP4 from supported versions`   should be 
`SUSE Linux Enterprise 12 SP3` etc, not the acronym of the product. 

on the  previous PR I missed that: `Add SLES 15 SP7 to supported versions (bsc#1245368)`

Reply:
for 5.1 it's ok. I put on hold the resubmission for now. if I can get it today, on time for the last updates round it's ok. if no, it will be included in the first maintenance update for 5.1.

Reply:
sorry for the back and forth and thanks for keeping pushing on this